internal/services/mysql/mysql_flexible_server_configuration_resource.go

```diff
@@ -20,8 +20,9 @@ import (
 
 func resourceMySQLFlexibleServerConfiguration() *pluginsdk.Resource {
 	return &pluginsdk.Resource{
-		Create: resourceMySQLFlexibleServerConfigurationCreate,
+		Create: resourceMySQLFlexibleServerConfigurationUpdate,
```

Comment:
Can we have separate Create and Update methods here and not re-use, we will need this going forward in any case.

Reply:
Sure. Updated.

![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/19754191/d4067a72-a461-427e-895a-81024cdcc7db)